### PR TITLE
Update consul exporter

### PR DESCRIPTION
**Pull request by limed**

### CODE REVIEW

**Comment by limed** [comment]
Fixes #302

File: nubis/puppet/consul_exporter.pp [hunk 1]
```diff
@@ -1,15 +1,24 @@
...
 
 notice ("Grabbing consul_exporter ${consul_exporter_version}")
 staging::file { "consul_exporter.${consul_exporter_version}.tar.gz":
   source => $consul_exporter_url,
 }->
 staging::extract { "consul_exporter.${consul_exporter_version}.tar.gz":
+  strip   =>  1,
   target  => '/usr/local/bin',
   creates => '/usr/local/bin/consul_exporter',
 }
```
Comment (gozer) [inline nubis/puppet/consul_exporter.pp]: Can't we just keep on using -> here, to keep things looking similar?

File: nubis/puppet/consul_exporter.pp [hunk 2]
```diff
@@ -1,15 +1,24 @@
...
 
+file { '/usr/local/bin/consul_exporter':
+  ensure  => file,
+  owner   => root,
+  group   => root,
+  mode    => '0755',
+  require =>  Staging::Extract["consul_exporter.${consul_exporter_version}.tar.gz"],
```
Comment (gozer) [inline nubis/puppet/consul_exporter.pp]: require => becomes unnecessary if we just used -> chaining above

Comment (limed) [reply nubis/puppet/consul_exporter.pp]: Fixed

**Pull request merged**